.travis.yml

```diff
@@ -8,6 +13,17 @@ go:
 
 install: go get -t ./...
 
+before_script:
+  # CodeClimate
+  - curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
+  - chmod +x ./cc-test-reporter
+  - ./cc-test-reporter before-build
```

Comment:
You realize you're running random code you download from the Internet here, right? :)